cdap-docs/developers-manual/source/building-blocks/workflows.rst

```diff
@@ -47,7 +47,7 @@ Executing MapReduce or Spark Programs
 -------------------------------------
 To execute MapReduce or Spark programs in a workflow, you will need to add them in your
 application along with the workflow. You can optionally add a :ref:`schedule <schedules>` 
-to the workflow using the `java api 
+to the workflow using the ``AbstractApplication`` `Java API 
```

Comment:
User app can directly implement the `Application` interface in which case the api he would use to add schedule would be this - https://github.com/caskdata/cdap/blob/develop/cdap-api/src/main/java/co/cask/cdap/api/app/ApplicationConfigurer.java#L101

Should we say instead that `You can optionally add schedule to the workflow using <link to ApplicationConfigurer#addSchedule api>. To add schedule to the application extending from AbstractApplication use <link to AbstractAppliactaion#sceduleWorklow api>`